assets/base/i18n/en.json

```diff
@@ -681,6 +681,7 @@
   "mobile.search.show_less": "Show less",
   "mobile.search.show_more": "Show more",
   "mobile.search.team.select": "Select a team to search",
+  "mobile.search.results": "{results} search results",
```

Comment:
after the code change, don't forget to update this

Reply:
updated!